js_functions/task2.js

```diff
@@ -0,0 +1,9 @@
+function identifyIfAdult(age) {
+ if (age >= 18) {
```

Comment:
nothing wrong with this code but it could be shorter
```js
function identifyIfAdult(age) {
    return age >= 18
}
```

Reply:
done